docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -0,0 +1,95 @@
+# Training models with GPU support
+:label:`sec_tabulargpu`
+
+Training with GPU can significantly speed up base algorithms, but it is also a necessity for text and vision models where even on small tasks and the best CPUs the training can take forever without GPU acceleration. To enable GPU training, pass `num_gpus` parameter into a `fit()` call. All models
+with GPU support will automatically start using the acceleration.
+
+```{.python}
+save_path = 'agModels-predictClass'
```

Comment:
Why save_path? This is unrelated to the tutorial

Reply:
removed